ledger-api/grpc-definitions/com/daml/ledger/api/v1/experimental_features.proto

```diff
@@ -23,6 +23,7 @@ message ExperimentalFeatures {
   ExperimentalContractIds contract_ids = 5;
   ExperimentalCommitterEventLog committer_event_log = 6;
   ExperimentalExplicitDisclosure explicit_disclosure = 7;
+  ExperimentalUserAndPartyManagementExtensionsForHub user_and_party_management_extensions_for_hub = 8;
```

Comment:
I propose dropping `ForHub` here… I don't see anything about the implementation that is truly Hub specific and none of these other flags so explicitly call out the team that is the source of the requirements.

Maybe `ExperimentalUserAndPartyLocalMetadataExtensions`?

Reply:
Ok, applied